case-lib/lib.sh

```diff
@@ -27,6 +27,9 @@ source "$SCRIPT_HOME/case-lib/hijack.sh"
     exit 2
 }
 
+# alsabat will return -2 for "C" locale
+export LC_ALL=en_US.UTF-8
```

Comment:
This change is good but wait: why is alsabat failing with a "C" locale and how? This sounds really bad. Please file a bug if none exists yet.

Reply:
> Please file a bug if none exists yet.

We already have https://github.com/intel-innersource/drivers.audio.ci.sof-framework/issues/286 but it is: 1. private 2. incomplete.